### ISSUE TITLE
report

### ISSUE THREAD

**Issue by wassimassi** [davidhaohanli/csc411proj#2]
is there any report or article that explain the work 
thanks 
best regards

**Comment by harshthakkar09** [comment]
Yes, if you can even list out references that would be helpful to us.
Thanking you.

**Comment by sanjayroka05** [comment]
Sir can you please provide the research paper or any document that explain the work in details.

**Comment by Belize9709** [comment]
Do you have any related papers for reference? Do you recommend it to me？
